### ISSUE TITLE
validation loss can not descend

### ISSUE THREAD

**Issue by JackChen-CS** [gary1346aa/Fast-RCNN-Pytorch#6]
Thank you for sharing :) I tried your code with 700 plane images, 
train loss can descend continuously, but validation loss can not descend after about 5 epochs.
Your readme notebook shows the same phenomenon, can you tell me how you overcome this problem ?
